tests/usm/usm_atomic_access.h

```diff
@@ -20,16 +20,21 @@
 
 namespace usm_atomic_access {
 
-static auto get_scalar_types() {
+inline auto get_nondouble_scalar_types() {
```

Comment:
Just curious about the role of inline here.

Reply:
`static` will cause warnings now that the different source files don't necessarily use the functions. Granted we may not need `inline` here, we will at least avoid duplicate definition errors if it ever becomes relevant.

Reply:
I see. Thanks.